### PR TITLE
improved location and votelog count

### PR DESCRIPTION
**Pull request by FedericoTartarini**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "fitbit": {
     "appType": "clockface",
-    "appDisplayName": "cozie",
+    "appDisplayName": "Cozie-fed-debugging",
```
Comment (pjayathissa) [inline package.json:4]: just make sure you don't update the app name and UUID

Comment (FedericoTartarini) [reply package.json]: I have changed that.

**Comment by FedericoTartarini** [comment]
@pjayathissa I have pushed all the latest changes I have implemented, I am not sure if you have review them all. 

Now the app is working perfectly, no more issues when a new participant is on-boarded. Cozie now works well also on the Versa Lite. The only issues that still persist, but cannot really be fixed are:
1. Cozie waits for 4 minutes to get GPS but that time is not always enough. It can be extended a bit but would not change much in my opinion. Consequently we do not always get GPS coorindates.
2. iPhone is not Fitbit friendly. It kills the app after a couple of days, when it is running in the background if the user do not interact with it. Hence, the user has to manually open the Fitbit app so data get synced. Not much we can do to fix this problem, plus since you added the q at the bottom of the screen the user is notified that the messages are not being pushed to the database.

Aside of these two issues that we do not have control over, I have tested this version of the app on both iOS and Android devices (multiple makers) and everything is working fine. Consequently I would invite you to: 

1. push these changes to the master branch;
2. update the images in the Fitbit gallery.

I will now work in the near future to bring in the new feature you have implemented (flow of surveys).

**Comment by FedericoTartarini** [comment]
Thank you very much for accepting the Pull request. No the 4 minute delay so far has not caused any issues. This version of the Cozie clock face is currently installed on the Fitbit of my 20 participants and it is working fine. I have extensively modified the code and planning to send a new Pull request soon. The objective was to clean the code, make it more readable and implement the flow you added to Cozie covid. So far it is working fine, I and Yi Ting are testing it.

**Comment by pjayathissa** [comment]
Awesome, excited to see it :)
Btw, in your refactoring, I was wondering if it was possible to split the main index.js into multiple smaller js files for easier reading?

**Comment by FedericoTartarini** [comment]
Yes that is what I have done. Created one file to handle the Buzz, one for updating the time, ..... etc. It is working fine the only issue that I am experiencing is with the buzzing.

**Pull request merged**